dcs-packaging-tool/dcs-packaging-tool-gui/src/main/resources/org/dataconservancy/config/applicationContext.xml

```diff
@@ -135,7 +137,21 @@
     <property name="localCountryCode" value="${localCountryCode}"/>
   </bean>
 
+  <bean id="emailValidator" class="org.dataconservancy.packaging.gui.util.EmailValidator" />
```

Comment:
In the interest of reserving some contracted hours for documentation work, this is something we can do on our side.